src/condor_credd/condor_credmon_oauth/examples/config/condor/40-oauth-credmon.conf

```diff
@@ -24,6 +24,16 @@ use feature : OAUTH
 # to execute hosts, if encryption is not already enabled.
 # SEC_DEFAULT_ENCRYPTION = REQUIRED
 
+# MANDATORY when requiring users to use the Vault credmon by default
+# Tell condor_submit to route token requests through htgettoken
+# SEC_CREDENTIAL_STORER = /usr/bin/condor_vault_storer
+
+# Set the time in seconds that credd will wait after jobs are
+# finished before deleting the user's credential directory. Don't
+# set this lower than 7 days because condor_vault_storer only stores
+# credentials in credd once a week.
+# SEC_CREDENTIAL_SWEEP_DELAY = 604800
```

Comment:
Without having looked at the rest of the PR, is there a good reason for this not to be enforced by the code?

Reply:
Credentials are marked for deletion when a user has no jobs in the queue. Right now, the credd has a timer that looks for mark files older than `SEC_CREDENTIAL_SWEEP_DELAY` and then removes the associated credentials. I don't think we want to duplicate this behavior in the credmon, if only because it makes debugging wrongfully deleted credentials more difficult (which is something we've run into before).

Reply:
I wasn't suggesting that we duplicate the behavior in the credmon.  I was asking why the code didn't enforce the minimum delay.  (For that matter, why are the code and param table defaults 3600 seconds?)